dagger-core/src/main/java/io/odpf/dagger/core/StreamManager.java

```diff
@@ -199,6 +197,10 @@ protected StreamInfo createStreamInfo(Table table) {
         return new StreamInfo(stream, table.getSchema().getFieldNames());
     }
 
+    List<Stream> getStreams() {
```

Comment:
any access modifier?

Reply:
it is protected, need this for mocking in tests